editor/gui/scene_tree_editor.h

```diff
@@ -168,6 +168,7 @@ class SceneTreeEditor : public Control {
 	void set_show_enabled_subscene(bool p_show) { show_enabled_subscene = p_show; }
 	void set_valid_types(const Vector<StringName> &p_valid);
 
+	void rename_node(Node *p_node, const String &p_name);
```

Comment:
I'm assuming this was a rebase mistake, there's already a `rename_node` in line 151.